swagger.yaml

```diff
@@ -5959,6 +5961,13 @@ definitions:
         type: integer
         description: 99th percentile of fsync duration in milliseconds
 
+  iscsi:
+    type: object
+    properties:
+      host_ip_address:
+        type: string
+        description: Host IP address used to reach iSCSI target
```

Comment:
@avishayt I plan to update the inventory API to add details specific to iSCSI drives, it would look like this:
```
{
  ...
  disks: [
    {
      name: "sda",
      drive_type: "iSCSI"
      iscsi: { # this is new
        host_ip_address: "..."   
      }                                         
    },
   {...}
] 
}
```

What do you think?

Reply:
OK by me